### PR TITLE
NAS-114753 / 22.02.1 / Add tests for NFS share parameters

### PR DESCRIPTION
**Pull request by anodos325**
This commit adds tests that verify that NFS
share parameters may be successfully set through the
REST API.

### CODE REVIEW

File: tests/api2/test_300_nfs.py
Comment (themylogin) [inline tests/api2/test_300_nfs.py:311]: The same path is exported twice, isn't it? `set` will have only one item so I don't understand how will this test ensure that we have two exports?

Comment (anodos325) [reply tests/api2/test_300_nfs.py:314]: Dumb oversight on my part. :)
I've updated to create two subdirectories and then add them both as paths (which is more in line with how this feature is designed anyway).

**Comment by anodos325** [comment]
Alright. I'll wait for it to finish separate run through ci (including new tests) to make sure it works in our CI infrastructure properly before merge.

**Review by themylogin — APPROVED** [review]

**Comment by anodos325** [comment]
CI passed.

**Pull request merged**
